src/pages/member/ProgramIssueCollectionAdminPage.tsx

```diff
@@ -142,7 +142,7 @@ const GET_MEMBER_PROGRAM_ISSUES = gql`
         app_id: { _eq: $appId }
         thread_id: { _like: $threadIdLike }
         solved_at: { _is_null: $unsolved }
-        issue_enrollment: { program: { program_enrollments: { member_id: { _eq: $memberId } } } }
+        issue_enrollment: { program: { program_content_enrollments: { member_id: { _eq: $memberId } } } }
```

Comment:
NITS: A little bit weird of using program content enrollment here
Although both program enrollment and program plan enrollment is in it